src/Explorer/Panes/SettingsPane/SettingsPane.tsx

```diff
@@ -829,6 +832,26 @@ export const SettingsPane: FunctionComponent<{ explorer: Explorer }> = ({
             </div>
           </div>
         )}
+        <div className="settingsSection">
+          <div className="settingsSectionPart">
+            <DefaultButton
+              onClick={() => {
+                useDialog
+                  .getState()
+                  .showOkCancelModalDialog(
+                    "Clear your settings and history",
+                    "Are you sure?",
+                    "Clear",
+                    () => deleteAllStates(),
+                    "Cancel",
+                    undefined,
+                  );
+              }}
+            >
+              Clear your settings and history
```

Comment:
```suggestion
                    "Clear local settings and history",
                    "This action will remove all custom configurations saved in your browser, such as filter history, personalized tab arrangements, and column settings in tables.",
                    "Clear",
                    () => deleteAllStates(),
                    "Cancel",
                    undefined,
                  );
              }}
            >
              Clear local settings and history
```
<img width="609" alt="image" src="https://github.com/user-attachments/assets/c7c52600-b1f1-49e2-ad3a-a511318c28a8">